wheatley/bot.py

```diff
@@ -111,7 +114,7 @@ def log_invalid_key(message: str) -> None:
     def _on_row_gen_change(self, row_gen_json: JSON) -> None:
         try:
             self.next_row_generator = json_to_row_generator(row_gen_json, self.logger)
-            self.logger.info("Successfully updated next row gen")
+            self.logger.info(f"Next time, Wheatley will ring {self.next_row_generator.summary_string()}")
```

Comment:
Could this more clearly reference it's updated the row generator for the next look to (as opposed to the next row). Is this only in server logs at the moment?

Reply:
Oh good point.  I think `Next touch, Wheatley will ....` will work better.